symfony/client/src/core/util/validation/rules.ts

```diff
@@ -182,3 +182,21 @@ export const decimalsOnly = function(value: string): boolean | string {
     'Should be a number'
   );
 };
+
+export const positiveNumber = function(value: string): boolean | string {
+  return (
+    value == '' ||
+    (/^\d+$/.test(value) && !Number.isNaN(parseFloat(value))) ||
+    'Should be a positive number'
+  );
+};
+
+export const lessthancharaters = function(power: number) {
+  return function(value: string): boolean | string {
+    return (
+      Number.isNaN(parseFloat(value)) ||
+      parseFloat(value) < Math.pow(10, power) ||
+      `Should be less than ${power} characters`
```

Comment:
is this correct?  `Should be less than 10 characters` if power 10

Reply:
no it's 10^2 , so when I give lessThanCharaters(2), and someone type 100, it display "Should be less than 2 characters"

Reply:
I'm also confused, maybe add a comment here. and why not use existing `shouldNotExceedCharLength` function?

Reply:
sure i will use that method